### PR TITLE
chore: reduce boilerplate and small fixups

### PR DESCRIPTION
**Pull request by cobraz**
I'm sorry for the large change. It should be easy enough to get through, as there's mostly deleted and moved files.

Let me know if I can explain anything better?

### CODE REVIEW

File: src/theme.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      green00: {
+        default: '#00837D',
+        _dark: '#C8FAFF',
+      },
+      green10: {
+        default: '#C8FAFF',
+        _dark: '#00837D',
+      },
+    },
+  },
```
Comment (braaar) [inline src/theme.tsx]: Which colours are we actually using?

File: src/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { chakra } from "@chakra-ui/react";
+import NextImage from "next/image";
+
+export const Image = chakra(NextImage, {
+  shouldForwardProp: prop => ['width', 'height', 'src', 'alt'].includes(prop),
+});
```
Comment (braaar) [inline src/utils.tsx:6]: Does it make sense to place this in the components folder?

**Review by braaar — APPROVED** [review]

File: src/theme.tsx (hunk 1 above)
Comment (cobraz) [reply src/theme.tsx]: Just reduced this down, I think we'll figure that one out later. https://github.com/veltolini/website/pull/20/commits/9a3fc13501940545be21039fe05a24c7503587f9

File: src/utils.tsx (hunk 2 above)
Comment (cobraz) [reply src/utils.tsx:6]: Maybe, yeah. Give me a few!

**Pull request merged**